modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -40,7 +40,7 @@ interface SelectedResizeHandle extends ResizeHandle {
 }
 
 const elementSelectionAttr = 'data-mce-selected';
-const controlElmSelector = 'table,img,figure.image,hr,video,span.mce-preview-object';
+const controlElmSelector = 'table,img,figure.image,hr,video,span.mce-preview-object,details';
```

Comment:
Could you add test for this change. There is a ControlSelectionTest

Reply:
Added the test for `data-mce-selected` attribute for details element.